### PR TITLE
Implement remaining draft IDs (seeking review)

### PR DESCRIPTION
**Pull request by JoelCFC25**
This adds a new function for all draft identities that sets each side's agenda point requirement to 6, per the draft format rules. Obviously this would cause a hiccup if someone chooses a draft ID against a non-draft ID, but hopefully there isn't too much of that happening, and @zaroth and I have discussed adding deck label indicator for Draft.

### CODE REVIEW

**Comment by Saintis** [comment]
Maybe you could extract the in-faction card counting function into one for each side with some fancy `defmulti` or so?

File: src/clj/game/cards-identities.clj [hunk 1]
```diff
@@ -114,6 +130,20 @@
...
+   "Fringe Applications: Tomorrow, Today"
+   {:effect draft-points-target
+    :events (let [fap {:req (req (let [facs (frequencies (map :faction (filter :rezzed (all-installed state :corp))))
+                                       hb (get facs "Haas-Bioroid" 0)
+                                       ji (get facs "Jinteki" 0)
+                                       nb (get facs "NBN" 0)
+                                       we (get facs "Weyland Consortium" 0)]
+                                   (and (> we ji) (> we hb) (> we nb))))
+                       :msg (msg "place 1 advancement token on " (card-str state target))
+                       :prompt "Choose a piece of ICE to place 1 advancement token on it" :player :corp
```
Comment (Saintis) [inline src/clj/game/cards-identities.clj]: If we are pedantic this should be "advancement counter" as per the card. But they are the same thing as ruled by Lucas.

Comment (JoelCFC25) [inline src/clj/game/cards-identities.clj]: FFG templating (null set) strikes again!

**Comment by Saintis** [comment]
Looks good!

**Comment by mtgred** [comment]
Is this ready for merging?

**Comment by JoelCFC25** [comment]
Yeah it's ready as-is. @Saintis can add his fancy `defmulti` thing later if he wants, I'm not sure I know how to do that.

**Comment by mtgred** [comment]
:+1:

**Pull request merged**